src/Common/Entity/Program.php

```diff
@@ -311,4 +312,20 @@ public function setEndDate($end_date): void
     {
         $this->end_date = $end_date;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getEnableEmailLogin()
+    {
+        return $this->enable_email_login;
+    }
+
+    /**
+     * @param mixed $enable_email_login
```

Comment:
mixed?  I'm pretty sure it should explode if it's not a 0 or a 1...

Reply:
all properties are left to mixed so I followed suit